crates/fluvio-cluster/src/runtime/local/k8s.rs

```diff
@@ -0,0 +1,211 @@
+use std::{
+    fs::File,
+    path::{PathBuf, Path},
+    process::{Command, Stdio},
+};
+
+use fluvio_command::CommandExt;
+use k8_config::{K8Config, User, UserDetail, Cluster, ClusterDetail, Context, ContextDetail};
+use rcgen::{Certificate, CertificateParams, IsCa, DistinguishedName, DnType, DnValue, KeyUsagePurpose};
+use tracing::{info, debug};
+
+use super::LocalRuntimeError;
+
+const DEFAULT_K8S_CONTEXT_NAME: &str = "fluvio-local";
+const DEFAULT_K8S_CLUSTER_NAME: &str = "fluvio-local-cluster";
+const DEFAULT_K8S_USER_NAME: &str = "fluvio-local-client";
+const DEFAULT_K8S_NAMESPACE: &str = "default";
```

Comment:
This should be `fluivo-system` or something like that?

Reply:
This k8s api will be dedicated to fluvio only and only one namespace will be used, so there will be no need for any separation, and the "default" is convenient, because if you want to look at it by `kubectl`, it is just less typing there, no need to type namespace name.

Reply:
fair enough